tools/sz_repo_cli/lib/src/commands/src/build_runner_build_command.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:async';
+
+import 'package:sz_repo_cli/src/common/common.dart';
+
+class BuildRunnerBuild extends ConcurrentCommand {
+  BuildRunnerBuild(SharezoneRepo repo) : super(repo) {
+    argParser.addFlag(
+      'delete-conflicting-outputs',
+      help:
+          'By default, the user will be prompted to delete any files which already exist but were not known to be generated by this specific build script.',
```

Comment:
> promoting the options works with our CLI

I don't know what do you mean exactly. Does that mean that you are unsure if we as developers would notice having that flag for the command?

I'm okay with defaulting it with true, since like you said that's the option we're always using anyways.

Reply:
Normally, when there is a conflict, the CLI prompts three options. You can then press key 1, 2 or 3.

```
[INFO] Found 25 declared outputs which already exist on disk. This is likely because the`.dart_tool/build` folder was deleted, or you are submitting generated files to your source repository.
Delete these files?
1 - Delete
2 - Cancel build
3 - List conflicts
```

However, because we don't show the prints from the Dart CLI, you will not see this message and you can't press any of these options.

Therefore, I would also hard-code `--delete-conflicting-outputs` to `true` because as a Sharezone Repo CLI user can't do anything else and wouldn't make sense to set `--delete-conflicting-outputs` to `false`.